packages/material-ui/src/TablePagination/TablePagination.test.js

```diff
@@ -233,33 +236,22 @@ describe('<TablePagination />', () => {
       assert.strictEqual(page, 0);
     });
 
-    it('should handle too high pages after changing rowsPerPage', () => {
-      let page = 2;
-      function ExampleTable(props) {
-        // setProps only works on the mounted root element, so wrap the table
-        return (
-          <table>
-            <TableFooter>
-              <TableRow>
-                <TablePagination
-                  count={11}
-                  page={page}
-                  onChangePage={(event, nextPage) => {
-                    page = nextPage;
-                  }}
-                  onChangeRowsPerPage={noop}
-                  {...props}
-                />
-              </TableRow>
-            </TableFooter>
-          </table>
-        );
-      }
-
-      const wrapper = mount(<ExampleTable rowsPerPage={5} />);
-      wrapper.setProps({ rowsPerPage: 10 });
-      // now, the third page doesn't exist anymore
-      assert.strictEqual(page, 1);
+    it('should raise a warning if the page prop is out of range', () => {
+      shallow(
+        <TablePagination
+          page={2}
+          rowsPerPage={5}
+          count={10}
+          onChangePage={noop}
+          onChangeRowsPerPage={noop}
+        />
+      );
+      assert.strictEqual(consoleErrorMock.callCount(), 1, 'should call console.error');
+      assert.match(
```

Comment:
You can use `include` here. It's what we use for the other warnings. No need to use a regex here.

Reply:
Actually, the tooltip (where I copied this from) uses a regex, too.

Reply:
I don't have a strong opinion about this. Since you already escaped the regex it doesn't really matter at this point anyway. Just a future tip since you don't actually use any features of regular expressions here as far as I can tell.